File: api/v2010/conference/update-example-announce_url/update-example-announce_url.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

conference = @client.conferences('CFbbe46ff1274e283f7e3ac1df0072ab39')
                    .update(announce_url: 'http://www.myapp.com/announce')

puts conference.friendly_name

File: api/v2010/conference/update-example-announce_url/update-example-announce_url.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var conference = ConferenceResource.Update(
            announceUrl: new Uri("http://www.myapp.com/announce"),
            pathSid: "CFbbe46ff1274e283f7e3ac1df0072ab39"
        );

        Console.WriteLine(conference.FriendlyName);
    }
}
